doc/man7/EVP_KDF_PBKDF2.pod

```diff
@@ -34,9 +34,21 @@ The supported controls are:
 
 These controls work as described in L<EVP_KDF_CTX(3)/CONTROLS>.
 
-B<iter> is the iteration count and its value should be greater than or equal to
-1. RFC 2898 suggests an iteration count of at least 1000.  The default value is
-2048.  Any B<iter> less than 1 is treated as a single iteration.
+=item B<EVP_KDF_CTRL_SET_PBKDF2_PKCS5_MODE>
+
+This control expects one argument: C<int checks>
+
+This control can be used to turn off the compliance checks. This can be used for
+PKCS5 related test vectors and scrypt which do not comply with SP800-132.
```

Comment:
I think that to retain feature compatibility (with strange/semi-broken deployments) Kerberos users of PBKDF2 will also need to set this option.  So I think rewording this to not sound like an exhaustive list would be wise.

Reply:
So you just want 
>This control can be used to turn off the compliance checks. This can be used for
>PKCS5 related test vectors and scrypt which do not comply with SP800-132.
reduced to..
This control can be used to turn off SP800-132 compliance checks.

We could also only enable this by default in fips mode? (But they do seem to be fairly valid checks though). We could also just not do the iteration check as well (as its only recommended - the salt however is a 'shall be statement')

Reply:
That would work, as would "can be used for cases such as [...]" or "This is needed in some cases, including but not limited to, [...]".